modules/swagger-codegen/src/main/resources/csharp/Project.mustache

```diff
@@ -88,7 +88,10 @@
     {{/netStandard}}
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs"/>
+    <Compile Include="Api\**\*.cs"/>
```

Comment:
@voydz Thank you, sir! And thanks for fixing the double serialization as well as exposing the `obj\*.cs` issue. Sorry for all the comments today, I just realized it is mid-week and it probably seems weird. I'm on paternity leave and had free time this morning for open source.